components/jira/actions/add-comment-to-issue/add-comment-to-issue.mjs

```diff
@@ -1,97 +1,71 @@
   key: "jira-add-comment-to-issue",
   name: "Add Comment To Issue",
-  description: "Adds a new comment to an issue.",
-  version: "0.1.1",
+  description: "Adds a new comment to an issue, [See the docs](https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-issue-comments/#api-rest-api-3-issue-issueidorkey-comment-post)",
+  version: "0.1.2",
   type: "action",
   props: {
-    jira: {
-      type: "app",
-      app: "jira",
-    },
+    jira,
     issueIdOrKey: {
-      type: "string",
-      description: "The ID or key of the issue where the comment will be added.",
-    },
-    expand: {
-      type: "object",
-      description: "The Jira REST API uses resource expansion, which means that some parts of a resource are not returned unless specified in the request. Use [expand](https://developer.atlassian.com/cloud/jira/platform/rest/v3/intro/#expansion) to include additional information about comments in the response. This parameter accepts `renderedBody`, which returns the comment body rendered in HTML.",
-      optional: true,
+      propDefinition: [
+        jira,
+        "issueIdOrKey",
+      ],
     },
     body: {
       type: "object",
+      label: "Body",
       description: "The comment text in [Atlassian Document Format](https://developer.atlassian.com/cloud/jira/platform/apis/document/structure/).",
     },
-    visibility_type: {
-      type: "string",
-      description: "Whether visibility of this item is restricted to a group or role.",
-      optional: true,
-      options: [
-        "group",
-        "role",
-      ],
-    },
-    visibility_value: {
-      type: "string",
-      description: "The name of the group or role to which visibility of this item is restricted.",
-      optional: true,
-    },
-    visibility_additional_properties: {
+    visibility: {
       type: "object",
-      description: "Extra properties of any type may be provided to the visibility object.",
+      label: "Visibility",
+      description: "The group or role to which this comment is visible, See `Visibility` section of [doc](https://developer.atlassian.com/cloud/jira/platform/rest/v3/api-group-issue-comments/#api-rest-api-3-issue-issueidorkey-comment-post)",
       optional: true,
     },
     properties: {
       type: "any",
+      label: "Properties",
       description: "A list of comment properties.",
       optional: true,
     },
-    additional_properties: {
+    additionalProperties: {
       type: "string",
+      label: "Additional properties",
       description: "Extra properties of any type may be provided to this object.",
       optional: true,
     },
+    expand: {
+      type: "string",
+      label: "Expand",
+      description: "The Jira REST API uses resource expansion, which means that some parts of a resource are not returned unless specified in the request. This parameter accepts `renderedBody`, which returns the comment body rendered in HTML.",
+      optional: true,
+    },
   },
```

Comment:
`properties`, `additionalProperties`, and `expand` are used in multiple components, so they should be moved to the app file. Then you'd use propDefinitions for them just like `issueIdOrKey`. You can overwrite the definitions in the components if they need to be different.

This applies to all props that are used in multiple components.

Reply:
I will do in this way, thanks!